newa/cli.py

```diff
@@ -101,10 +101,10 @@ def cmd_event(ctx: CLIContext, errata_ids: tuple[str, ...]) -> None:
             event = Event(type_=EventType.ERRATUM, id=erratum_id)
```

Comment:
Does this `_` belong here? In the YAML it produces records like:
```
event:
  id: '128049'
  type_: erratum
```

Reply:
`type` is a Python built-in function, hence `type_` to avoid the conflict. And it's ugly and I'd be happy to rename the field to something better.

Reply:
I agree